website/docs/r/static_site.html.markdown

```diff
@@ -32,6 +32,10 @@ The following arguments are supported:
 
 * `resource_group_name` - (Required) The name of the Resource Group where the Static Web App should exist. Changing this forces a new Static Web App to be created.
 
+* `sku_tier` - (Required) The Sku of the static site ("Free" or "Standard")
```

Comment:
According to schema sku_tier and sku_size are optional and default to Free.

 (Optional) Specifies the sku tier of the Static Web App. Possible values are "Free" or "Standard". Defaults to "Free".